docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -18,23 +18,25 @@ To make your validator node and validator fullnode deployment hassle-free, make
 ## Hardware requirements
 
 For running an Aptos **validator node and validator fullnode** we recommend the following hardware resources:
+For running a validator and validator fullnode is that your hardware spec should be good enough to achieve a 30k peak
+TPS on mainnet. In order to evaluate that, you can use the command (https://gist.github.com/igor-aptos/298c87c0a5f095bdae4716b526d82520)
+It has two benchmarks, a shorter one (~10m) and a large db one (few hours). Once the script finishes, it will print out a table like: https://gist.github.com/igor-aptos/97d43db4f460ab4b29e4902cb3d31954, and column "t/s" is your achieved "transactions per second". Evaluation criteria is encoded in the script, and will warn you after the table if you've not met any of the criteria.
 
-- **CPU**:
-  - 8 cores, 16 threads
+If you don't want to run the benchmark tool for hardware evaluation, you can use the following reference specs
+
+- **CPU Requirement Reference**:
+  - 32 cores
   - 2.8GHz, or faster
-  - Intel Xeon Skylake or newer
-- **Memory**: 32GB RAM.
-- **Storage**: 2T SSD with at least 40K IOPS and 200MiB/s bandwidth.
+  - AMD Milan EPYC or Intel(R) Xeon(R) Platinum
+- **Memory**: 64GB RAM.
+- **Storage**: 2T SSD with at least 60K IOPS and 200MiB/s bandwidth.
 - **Networking bandwidth**: 1Gbps
-
-### Example machine types on various clouds
-
-- **AWS**
-  - c6id.8xlarge (if using a local SSD)
-  - c6i.8xlarge + io1/io2 EBS volume with 40K IOPS.
-- **GCP**
-  - n2-standard-16 (if using a local SSD)
-  - n2-standard-32 + pd-ssd with 40K IOPS.
+- **Example machine types on various clouds**:
+  - **AWS**
+    - c6id.16xlarge (if using a local SSD)
+    - c6i.16xlarge + io1/io2 EBS volume with 60K IOPS.
```

Comment:
On GCP, our recommendation is for the `N2/N2D/T2D` families, which have RAM:CPU of 4:1. Shouldn't we recommend the same on AWS as well, i.e. the `M6I/M6A` instead of `C6I/C6A` which only have 2:1 RAM:CPU ratio ?

Reply:
We are not really bottlenecked on RAM - 128 GB of RAM is more than enough for running at 30k TPS. What is the cost difference between `M6I/M6A` vs C6I/C6A ?

Reply:
10% more for same vCPU count.

Reply:
what about the number of physical cores?

Reply:
Monthly prices:
c6a.16xlarge(64/128G): $1787
c6i.16xlarge(64/128G): $1985
m6a.16xlarge(64/256G): $2018
m6i.16xlarge(64/256G): $2242

On all the instance types, a vCPU is a hyperthread. AWS doesn't have instances where a vCPU is a full core, like `T2D`.

Reply:
Since the `c6` family is cheaper and we are not bottlenecked by the RAM, let's keep the `c6` family as reference spec?